diff --git a/CMakeLists.txt b/CMakeLists.txt
index 53c9d98..80f0a77 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -13,7 +13,7 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
     set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/release" CACHE PATH "Default install path" FORCE)
 endif()
 # set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib ${CMAKE_SOURCE_DIR}/lib/lib)
-set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
diff --git a/cmake/dxrt.function.cmake b/cmake/dxrt.function.cmake
index 8f6fdec..fbcc325 100644
--- a/cmake/dxrt.function.cmake
+++ b/cmake/dxrt.function.cmake
@@ -52,5 +52,4 @@ macro(add_target target)
     add_executable(${target} ${ARG_SRC_LIST})
     add_dxrt(${target})
     install(TARGETS ${target} DESTINATION bin)
-    install(TARGETS ${target} DESTINATION ${CMAKE_SOURCE_DIR}/bin)
 endmacro(add_target)
