diff --git a/CMakeLists.txt b/CMakeLists.txt
index ed1cfa4..85f3ab6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -25,7 +25,7 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
     set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/release" CACHE PATH "Default install path" FORCE)
 endif()
 # set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib ${CMAKE_SOURCE_DIR}/lib/lib)
-set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib ${CMAKE_BINARY_DIR}/lib)
+set(CMAKE_INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/lib)
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
@@ -249,4 +249,4 @@ foreach(variable ${variables})
     if(variable MATCHES "^USE_" OR variable MATCHES "^ENABLE_")
         message(STATUS "    ${variable}: ${${variable}}")
     endif()
-endforeach()
\ No newline at end of file
+endforeach()
diff --git a/cmake/dxrt.function.cmake b/cmake/dxrt.function.cmake
index 653f937..8f29fa7 100644
--- a/cmake/dxrt.function.cmake
+++ b/cmake/dxrt.function.cmake
@@ -57,5 +57,4 @@ macro(add_target target)
     add_executable(${target} ${ARG_SRC_LIST})
     add_dxrt(${target})
     install(TARGETS ${target} DESTINATION bin)
-    install(TARGETS ${target} DESTINATION ${CMAKE_SOURCE_DIR}/bin)
 endmacro(add_target)
