diff --git a/cmake/dxrt.cfg.cmake b/cmake/dxrt.cfg.cmake
index a4d72ec..d8beb37 100644
--- a/cmake/dxrt.cfg.cmake
+++ b/cmake/dxrt.cfg.cmake
@@ -9,6 +9,6 @@ else()
 endif()
 option(USE_DXRT_TEST "Use DXRT Unit Test" ON)

-option(USE_ORT "Use ONNX Runtime" ON)
+option(USE_ORT "Use ONNX Runtime" OFF)
 option(USE_PYTHON "Use Python" ON)
-option(USE_SERVICE "Use DXRT Service" ON)
+option(USE_SERVICE "Use DXRT Service" OFF)



