From 8fc003785db35ecad6bdb9db61330c7fd6b79869 Mon Sep 17 00:00:00 2001
From: Andreas Rebert <andreas.rebert@embeddedartists.com>
Date: Mon, 15 Dec 2025 13:43:56 +0000
Subject: [PATCH 2/3] Change path to gen.h

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 40b3d91..3c203d9 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,7 +72,7 @@ include(dxapp.function)
 
 if(CROSS_COMPILE)
     set(CMAKE_SKIP_RPATH TRUE)
-    file(READ "${DXRT_INSTALLED_DIR}/include/dxrt/gen.h" header_contents)
+    file(READ "${DXRT_GEN_FILE_ROOT_DIR}/include/dxrt/gen.h" header_contents)
     if(header_contents MATCHES "#define USE_ORT")
         message(STATUS "THIS IS ONNX_RUNTIME")
         set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -Wl,-rpath-link,${onnxruntime_LIB_DIRS}")
-- 
2.34.1

